.github/workflows/e2e-test.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
           - arch: amd64
             # nb: use format(..) to catch both inputs.push-yamls = true AND inputs.push-yamls = 'true'.
             push-yamls: ${{ inputs.push-yamls }}
+        arch: [ amd64, arm64 ]
```

Comment:
IIUC this is no longer correct, as of 5bd47d245642b58d9f3c28617fc19bf03376c6b7; see the "TODO: when adding arm64 ..." above